### PR TITLE
Update free-programming-interactive-tutorials-en.md

### PR DESCRIPTION
**Pull request by Snimer**
Added many tutorials by w3schools

## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by SethiPandi** [comment]
Could you please make the additions more consistent?

You've written the resources in 3 different ways despite all of them coming from the same place.

I'd vouch just `[Xyz Tutorial]({link}) - W3Schools` would be fine for all of them.

Also could we write W3Schools with the correct capitalization.

**Comment by Snimer** [comment]
> Could you please make the additions more consistent?
> 
> You've written the resources in 3 different ways despite all of them coming from the same place.
> 
> I'd vouch just `[Xyz Tutorial]({link}) - W3Schools` would be fine for all of them.
> 
> Also could we write W3Schools with the correct capitalization.

Thank you for your suggestion.

File: free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -150,6 +153,7 @@
 * [Functional Programming in Javascript](https://github.com/ReactiveX/learnrx)
 * [Javascript interactive tutorial on CodeCademy](https://www.codecademy.com/learn/javascript)
 * [JavaScript interactive tutorial on CoderMania](http://www.codermania.com/javascript/lesson/1a/hello-world)
+* [Javascript Tutorial](https://www.w3schools.com/js/default.asp) - W3Schools
```
Comment (SethiPandi) [inline free-programming-interactive-tutorials-en.md:156]: Could we also make JavaScript the correct capitalization as well?

```suggestion
* [JavaScript Tutorial](https://www.w3schools.com/js/default.asp) - W3Schools
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Thank you for making those changes, looks great!

Sorry, I just reviewed again and noticed I missed something.

Could we just correct the capitalization of another noun?

File: free-programming-interactive-tutorials-en.md [hunk 2]
```diff
@@ -150,6 +153,7 @@
 * [Functional Programming in Javascript](https://github.com/ReactiveX/learnrx)
 * [Javascript interactive tutorial on CodeCademy](https://www.codecademy.com/learn/javascript)
 * [JavaScript interactive tutorial on CoderMania](http://www.codermania.com/javascript/lesson/1a/hello-world)
+* [JavaScript Tutorial](https://www.w3schools.com/js/default.asp) - W3Schools
```
Comment (eshellman) [inline free-programming-interactive-tutorials-en.md:156]: maybe use the same url format hare as for the others?

**Pull request merged**